lib/factory_bot/syntax/methods.rb

```diff
@@ -129,6 +129,20 @@ def generate_list(name, count)
           Internal.sequence_by_name(name).next
         end
       end
+
+      # Creates a copy of the file passed to attach to the attribute
+      #
+      # Arguments:
+      #   filepath: (String)
+      #     The path for the file that will be loaded. Should be a valid path
+      #     for a file that exists at that location.
+      #
+      # Returns:
+      #   An instance of the file loader class pointing to the file at the
+      #   provided path.
+      def file_fixture(file_path)
```

Comment:
This should go on the definition proxy, since it only needs to be available inside the attribute block. Otherwise when people include these syntax methods into their test suite it will end up overwriting the original ActiveSupport `file_fixture` method.

Reply:
Good idea, we don't want to overwrite the original ActiveSupport file_fixture method.

But we also need to be able to call file_fixture from transient attributes and callbacks...